src/main/sh/mommy

```diff
@@ -350,6 +350,7 @@ while getopts ":hv1d:c:e:s:-:" OPT; do
     c|config) opt_config="$OPTARG" ;;
     e|eval) require_arg; opt_eval="$OPTARG" ;;
     s|status) require_arg; require_int; opt_status="$OPTARG" ;;
+    r|rename) require_arg; opt_rename="$OPTARG" ;;
```

Comment:
for clarity, could you rename the variable to `opt_rename_to`? that way, it's clear that it contains the new name, not the old name.